bitnami/rabbitmq-cluster-operator/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: rabbitmq-cluster-operator
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq-cluster-operator
   - https://github.com/rabbitmq/cluster-operator
-version: 2.0.9
+version: 2.0.10
```

Comment:
Please bump the minor version instead since you're adding a new parameter.

Reply:
Bumped to next minor version